### DISCUSSION TITLE
With German keyboard layout, the shortcut `/` for search does not work.

### DISCUSSION THREAD

**Discussion by rillig** [community/community#59043] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

On a German keyboard, press Shift+7, which results in the `/` character.
Nothing happens.

Knowing that on an English keyboard layout, the `/` is on a key in the lower right area, press that key.
Nothing happens.

**Comment by leonden** [comment]
That's actually quite a common case. Personally I experience this particular issue pretty often. Most services that include some sort of user input by keys, are optimized for the US layout. I use the Swiss German and the US layout depending on what I work on. My keyboard though, is ISO US because for me it feels just much more convenient to write code.

**Comment by gabel** [comment]
~~Any known workaround for this one?~~

Use `s` key to open search.

**Comment by trkoch** [comment]
`s` does not work for me. It opens the settings instead.

**Comment by MoaazHF** [comment]
The root cause is usually that the application is listening for a specific physical key code (often the location of / on a US keyboard) rather than the actual character produced, or it fails to account for the modifier keys (like Shift) required to produce that character on different layouts.

US Layout (QWERTY): The / is a primary key (usually next to the Right Shift). Pressing it triggers a specific keyCode (often 191) or event.code ("Slash").

German Layout (QWERTZ): The / character requires holding Shift + 7.

The Failure: The code likely listens for the "Slash" key press without modifiers. On a German keyboard, pressing Shift + 7 triggers the "Digit7" code with a Shift modifier. If the app explicitly forbids modifiers for the shortcut (to avoid conflict with ?), it ignores the input.

**Comment by clementaudic** [comment]
Same problem with an AZERTY layout (French).
